test_conformance/extensions/cl_khr_command_buffer/command_buffer_out_of_order.cpp

```diff
@@ -33,7 +33,7 @@ struct OutOfOrderTest : public BasicCommandBufferTest
                    cl_command_queue queue)
         : BasicCommandBufferTest(device, context, queue),
           out_of_order_command_buffer(this), out_of_order_queue(nullptr),
```

Comment:
@EwanC  I just noticed this one `out_of_order_command_buffer(this)` copied from former test, looks like some misprint or maybe there is some deeper sense behind ?

Reply:
Good question, the [clCommandBufferWrapper](https://github.com/KhronosGroup/OpenCL-CTS/blob/main/test_conformance/extensions/cl_khr_command_buffer/command_buffer_test_base.h#L87-L91) class takes a `CommandBufferTestBase *` in the constructor which the other wrapper classes don't. So passing `this` is intentional here.